apim/values.yaml

```diff
@@ -257,6 +257,16 @@ api:
     pullPolicy: Always
   # pullSecrets:
   #   - name: gravitee_secrets
+  ssl:
+    enabled: false
+#    keystore:
+#      path: ${gravitee.home}/security/keystore.jks
+#      password: secret
+    clientAuth:
```

Comment:
I would prefer to keep the same structure we can have in `gravitee.yml` https://github.com/gravitee-io/gravitee-gateway/blob/1.30.x/gravitee-gateway-standalone/gravitee-gateway-standalone-distribution/src/main/resources/config/gravitee.yml#L27

Or maybe you have a good reason to change it?